.github/workflows/pull_request.yml

```
@@ -0,0 +1,38 @@
+name: Build
+
+on: 
+  pull_request:
+    types:
+      [opened, synchronize]
+    
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    container:
+      image: hulkhan/bannerlordcoop:1.0.2
+      credentials:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+      
+    steps:
+      - name: Checkout Repo
+        run: |
+          git clone https://github.com/${{ github.repository }}
+          cd BannerlordCoop
+          git fetch origin "pull/${{ github.event.number }}/head:${{ github.event.pull_request.base.ref }}-PR-${{ github.event.number }}"
+          git checkout "${{ github.event.pull_request.base.ref }}-PR-${{ github.event.number }}"
+
+      - name: Link Game Assemblies Folder
+        run: |
+          ln -s /home/mb2 BannerlordCoop
+          
+      - name: Build Solution
```

Also create another build and test for the Common project

```suggestion
      - name: Build Coop.Core
```